ee/ucp/authorization/create-teams-with-ldap.md

```diff
@@ -6,28 +6,25 @@ keywords: authorize, authentication, users, teams, UCP, LDAP
 
 >{% include enterprise_label_shortform.md %}
 
-To enable LDAP in UCP and sync to your LDAP directory:
+If Docker Enterprise is configured to sync users with your organization's Lightweight Directory Access Protocol (LDAP) directory server, you can enable syncing the new team's members when creating a new team or when modifying settings of an existing team.
```

Comment:
Suggest editing to read: "If Docker Enterprise is configured to sync users with a Lightweight Directory Access Protocol (LDAP) directory server, you can enable the syncing of new team members to occur either when creating a new team or when modifying settings of an existing team."